cfgov/unprocessed/css/molecules/global-search.less

```diff
@@ -267,6 +258,22 @@
     } );
 }
 
+.m-global-search {
+    // Tab trigger is used to capture press of the tab key so that
+    // global search can be collapsed when it hits this element.
```

Comment:
I like it, it's a smart work around to a wicked complicated set of interactions. Nice work!